src/content/docs/synthetics/synthetic-monitoring/getting-started/new-runtime.mdx

```diff
@@ -0,0 +1,100 @@
+---
+title: Convert to the new synthetic runtimes
+tags:
+  - Synthetics
+  - Synthetic monitoring
+  - Runtime conversion
+metaDescription: "Synthetic monitoring's new runtime requires that you make some changes to your already existing monitors."
+---
+
+We're introducing a new runtime for public monitoring locations. We plan to apply the new runtime to private monitors by the end of 2022. While we're providing backwards support for monitors using the Legacy runtime and its features, we recommend converting to the current runtime as soon as possible. 
```

Comment:
I'd love for the first sentence to set the stage a little more clearly and also maybe explain the reason for this change. (You may need to check with SME?)

Something like:  "Synthetic monitoring  has a new runtime to monitor public locations. This new runtime does X. We recommend converting your legacy runtime to take advantage of these new features." Something like that. I'm super hazy on the details.

If at all possible, I prefer to avoid future planning statements with dates (people tend to read these in docs as guarantees). So, instead, if unavoidable: "Eventually, we plan to make this new runtime available for private monitoring locations."